Maxis/Infrastructure/Repositories/UserRepository.cs

```diff
@@ -60,91 +53,79 @@ public List<EditUserViewModel> SelectById(long id)
                             UserId = ep.UserId,
                             Username = ep.Username,
                             Email = ep.Email,
+                            Mobile = ep.Mobile,
                             Department = ep.Department,
                             Title = ep.Title,
                             Status = ep.Status,
                             Roles = e.RoleName
-                        }).ToList();
+                        }).Single();
                      
-                    return (result.ToList());
-                
+                    return (result); 
             }
             catch (Exception )
             {
                 throw;
             }
         }
-
-        /// <summary>
-        /// 
-        /// </summary>
-        /// <param name="model"></param>
-        /// <returns></returns>
-        public IQueryable Insert(LoginViewModel loginViewModel)
+        public LoginViewModel Insert(LoginViewModel loginViewModel)
         {
             try
             {
-                   var user = _db.ONNET_USER.FirstOrDefault(u => u.Username == loginViewModel.Username && u.Password == loginViewModel.Password);
+                var user = _db.ONNET_USER.FirstOrDefault(u => u.Username == loginViewModel.Username && u.Password == loginViewModel.Password);
 
                     if (user == null)
                     {
                         var newuser = new ONNET_USER
                         {
                             Username = loginViewModel.Username,
-                            Password = loginViewModel.Password,
+                            //Password = Encrypt(new  LoginViewModel
+                            //{
+                                Password =loginViewModel.Password,
+                            //}),
                             Email = null,
```

Comment:
i dont think we need to set null value if DB column is set with default or isnull property